instrumentation/java-concurrent/java-completablefuture/java-completablefuture.gradle

```diff
@@ -0,0 +1,6 @@
+// Set properties before any plugins get loaded
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
```

Comment:
Maybe instead of creating a whole new module just for one test, put it with all others but guard with `spock.lang.Requires` check for current java version?

Reply:
I am not sure. We compile everything with java11, test’s bytecode will be 7 but it will use a class absent from java7 sdk. If Spock will run it only on 8+ everything should work.

But at least I think we have to document why this separate module exists. Then we will remember to merge it back when/if we abandon java7

Reply:
oh what you are suggesting works because it's groovy, so it isn't subject to javac `--release 7` restrictions 😭

i will implement your suggestion 👍

Reply:
done

Reply:
Aaa! I was wrong in my line of thought, `-release 7` would break my suggestion indeed. But anyway, all good that ends good